server/config/config.go

```diff
@@ -86,19 +88,42 @@ type LoggingConfig struct {
 	DisableBanner bool `yaml:"disable_banner"`
 }
 
+// AwsConfig defines configs related to AWS services
+type AwsConfig struct {
```

Comment:
How do you feel about also putting these under firehoseconfig struct? Once again, for the same reason, perhaps the community introduces a separate orthogonal thing related to AWS that also requires creds (stilly straw-man example: pulling in pack files stored in s3) they may want to use different creds.

What do you think?

Reply:
I would suggest that it's typically going to be easier to use a single service account with the appropriate permissions.

But I would like to avoid having to do any further deprecation of configs in the future so I will break these out directly into the `FirehoseConfig`.